setup.py

```diff
@@ -40,7 +40,7 @@
         'threadloop>=1,<2',
         'thrift',
         'tornado>=4.3,<5',
-        'opentracing>=1.2.2,<2',
+        'opentracing>=2.0,<3.0',
```

Comment:
pleasantly surprised that crossdock test works with `'opentracing_instrumentation>=2,<3'`